gravatar-quickeditor/src/main/res/values/available_languages.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--Warning: Auto-generated file, do not edit.-->
 <resources xmlns:tools="http://schemas.android.com/tools">
-  <string-array name="available_languages" translatable="false" tools:ignore="InconsistentArrays, UnusedResources">
+  <string-array name="available_languages" translatable="false" tools:ignore="InconsistentArrays">
```

Comment:
@wzieba @iangmaia 
Unfortunately, this is being overwritten when the translations from Glotpress are pulled. I will update this manually but if you have any idea how to automate this it would be great! 

[Previous discussion for context](https://github.com/Automattic/Gravatar-SDK-Android/pull/315#discussion_r1765226264)